bpf/lib/policy.h

```diff
@@ -66,18 +70,18 @@ __policy_can_access(void *map, struct __sk_buff *skb, __u32 identity,
 		.pad = 0,
 	};
 
-#ifdef HAVE_L4_POLICY
-	policy = map_lookup_elem(map, &key);
-	if (likely(policy)) {
-		cilium_dbg3(skb, DBG_L4_CREATE, identity, SECLABEL,
-			    dport << 16 | proto);
-
-		/* FIXME: Use per cpu counters */
-		__sync_fetch_and_add(&policy->packets, 1);
-		__sync_fetch_and_add(&policy->bytes, skb->len);
-		goto get_proxy_port;
+	if (HAVE_L4_POLICY && !is_fragment) {
```

Comment:
Maybe we should get rid of `HAVE_L4_POLICY` entirely?

Reply:
I expect that this will be a subsequent follow-up given that it's one of the causes for recompilation at the moment.

Reply:
Makes sense